src/backup_camera.cpp

```diff
@@ -126,16 +128,22 @@ bool BackupCamera::process_events()
                         signalToQuit();
                         return false;
                         break;
+
+                    case SDLK_f:
+                        printf("f was pressed: toggle fullscreen\n");
+                        return 2;
```

Comment:
You're mixing up returning ints and bools in this function, pretty confusing to read. Also this `2` is a magic number, it should be encased as a variable.